firedrake/utility_meshes.py

```diff
@@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, L):
```

Comment:
It is, but the method of periodic rectangle mesh doesn't support less than
3 elements wide, and the method of this new code doesn't easily extend to
the general case.
Cjc

On Wednesday, 8 June 2016, David A. Ham <notifications@github.com> wrote:

> In firedrake/utility_meshes.py
> <https://github.com/firedrakeproject/firedrake/pull/803#discussion_r66275587>
> :
>
> > @@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
> >      return PeriodicIntervalMesh(ncells, length=1.0)
> >
> >
> > +def OneElementThickMesh(ncells, L):
>
> Isn't this just a special case of PeriodicRectangleMesh?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/firedrake/pull/803/files/0bbb07b2ed6725ca0ceca05250a2adc5380d3632#r66275587>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0q8v8J99ng5FtKKW7nOgIYu9HFpuj4ks5qJtyBgaJpZM4Iw-Dk>
> .
>


-- 
http://www.imperial.ac.uk/people/colin.cotter

www.cambridge.org/9781107663916